docs/changelog.rst

```diff
@@ -2,6 +2,7 @@
 Changelog
 =========
 
+* :support:`245` Add expected propagate_properties behavior tests
 * :bug:`244` Revert Protocol.propagate_properties to use Well.add_properties
```

Comment:
something to note here is that we probably want a `major` flag attached to `244` unless we want an intermediate bugfix release
i..e :bug:`244 major`